src/Compilers/CSharp/Portable/Symbols/Synthesized/ReadOnlyListType/SynthesizedReadOnlyListTypeSymbol.cs

```diff
@@ -0,0 +1,576 @@
+
+        internal override bool IsFileLocal => false;
+
+        internal override FileIdentifier? AssociatedFileIdentifier => null;
+
+        internal override bool MangleName => false;
+
+        internal override bool HasDeclaredRequiredMembers => false;
+
+        internal override bool HasCodeAnalysisEmbeddedAttribute => false;
+
+        internal override bool IsInterpolatedStringHandlerType => false;
+
+        internal override bool HasSpecialName => false;
+
+        internal override bool IsComImport => false;
+
+        internal override bool IsWindowsRuntimeImport => false;
+
+        internal override bool ShouldAddWinRTMembers => false;
+
+        internal override TypeLayout Layout => default;
+
+        internal override CharSet MarshallingCharSet => DefaultMarshallingCharSet;
+
+        internal override bool HasDeclarativeSecurity => false;
+
+        internal override bool IsInterface => false;
+
+        internal override NamedTypeSymbol? NativeIntegerUnderlyingType => null;
+
+        internal override NamedTypeSymbol BaseTypeNoUseSiteDiagnostics => ContainingAssembly.GetSpecialType(SpecialType.System_Object);
+
+        internal override bool IsRecord => false;
+
+        internal override bool IsRecordStruct => false;
+
+        internal override ObsoleteAttributeData? ObsoleteAttributeData => null;
+
+        public override ImmutableArray<Symbol> GetMembers() => _members;
+
+        public override ImmutableArray<Symbol> GetMembers(string name) => GetMembers().WhereAsArray(m => m.Name == name);
+
+        public override ImmutableArray<NamedTypeSymbol> GetTypeMembers() => ImmutableArray<NamedTypeSymbol>.Empty;
+
+        public override ImmutableArray<NamedTypeSymbol> GetTypeMembers(ReadOnlyMemory<char> name, int arity) => ImmutableArray<NamedTypeSymbol>.Empty;
+
+        public override ImmutableArray<NamedTypeSymbol> GetTypeMembers(ReadOnlyMemory<char> name) => ImmutableArray<NamedTypeSymbol>.Empty;
+
+        protected override NamedTypeSymbol WithTupleDataCore(TupleExtraData newData) => throw ExceptionUtilities.Unreachable();
+
+        internal override NamedTypeSymbol AsNativeInteger() => throw ExceptionUtilities.Unreachable();
+
+        internal override ImmutableArray<string> GetAppliedConditionalSymbols() => ImmutableArray<string>.Empty;
+
+        internal override AttributeUsageInfo GetAttributeUsageInfo() => default;
+
+        internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<TypeSymbol> basesBeingResolved) => BaseTypeNoUseSiteDiagnostics;
+
+        internal override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(ConsList<TypeSymbol> basesBeingResolved) => ImmutableArray<NamedTypeSymbol>.Empty;
+
+        internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers() => GetMembersUnordered();
+
+        internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers(string name) => GetMembers(name);
```

Comment:
Are either of these actually going to get called?

Reply:
No, updated to throw `Unreachable()` exceptions.